src/objects/table/table.scss

```diff
@@ -0,0 +1,84 @@
+@use "../../design-tokens/colors.yml";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+
+$table-border-color: colors.$gray-light;
+$table-border-thin: sizes.$edge_small solid $table-border-color;
+$table-border-thick: sizes.$edge_medium solid $table-border-color;
+
+/**
+ * 1. Override some table defaults
+ * 2. Tables may overflow horizontally. Keep this from breaking layouts while
+ *    ensuring content is viewable
+ *    In the future we may want to consider making it visually obvious when
+ *    tables are clipped.
+ * 3. Override default `th` text alignment
+ */
+
+.o-table {
+  border-collapse: collapse; /* 1 */
+  border-spacing: 0; /* 1 */
+  overflow-x: auto; /* 2 */
+  text-align: left; /* 3 */
+  width: 100%; /* 2 */
+}
+
+/**
+ * Table cells and headers share padding
+ */
+.o-table__cell,
+.o-table__heading {
+  padding: ms.step(-3);
+}
+
+/**
+ * Table headings are bold to stand out as labels
+ */
+.o-table__heading {
+  font-weight: font.$weight-bold;
+}
+
+/**
+ * Table headers and footers are separated from the table body with strokes.
+ */
+.o-table__header {
+  border-bottom: $table-border-thick;
+}
+
+.o-table__footer {
+  border-top: $table-border-thick;
+}
+
+/**
+ * Table rows can be styled in a couple ways.
+ * This is achieved by adding a class to the table itself.
+ */
+.o-table--ruled .o-table__row {
+  border-bottom: $table-border-thin;
+}
+
+.o-table--striped .o-table__row:nth-of-type(2n - 1) {
+  background: colors.$gray-lighter;
```

Comment:
Is it necessary to override all of `background`?

Reply:
No, good call. I can use background-color

Reply:
Done!